web/styles/popovers.css

```diff
@@ -1207,15 +1207,22 @@ ul {
     border: solid 1px var(--color-border-dropdown-menu);
     background-color: var(--color-background-dropdown-menu);
     max-height: 85vh;
-    min-width: 230px;
     overflow-x: hidden;
     user-select: none;
     border-radius: 6px;
     box-shadow: var(--box-shadow-navbar-dropdown-menu);
 
     .simplebar-content {
+        min-width: 230px;
```

Comment:
I don't have the reasoning behind this value, since that wasn't introduced via this PR. I did some exploration and found the origin of this value [here](https://github.com/Ddharmani3/zulip/blob/be1902477a72871a700bd922221fa6c07e7231ce/web/styles/zulip.css#L3487).

Reply:
Got it. Well, as I pointed out in #28092, it might be best to just make this a descriptive variable (`--min-width-popover-something-something`?) for the places where its shared.